### PR TITLE
Flow extra type definition for camera callbacks

### PR DESCRIPTION
**Pull request by rfreitas**
Extra type definition on callbacks for camera

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-284-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-284-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://284-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Camera/index.js [hunk 1]
```diff
@@ -76,14 +76,13 @@ type CameraType = {
...
-
 }
 
 export default class Camera extends React.Component<CameraType> {
 
-  webcam: ?Webcam = null
+  webcam: Webcam = null
   interval: ?Visibility
-  countdown: ?CountUp
+  countdown = null
```
Comment (hjheath) [inline src/components/Camera/index.js]: Why do we need this?

Comment (rfreitas) [reply src/components/Camera/index.js]: I think is so that Flow picks up that this is a property and it can be null

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-284-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://284-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-284-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**